openstack/nova/compute.sls

```diff
@@ -37,6 +35,8 @@ add libvirt-qemu to nova:
     - delusers:
       - libvirt-qemu
 
+{% if not virl.mitaka %}
```

Comment:
I've kept kilo.nova.conf as is, hence am not sure how necessary this is. For mitaka, the nova.conf jinja template should take care of everything. This is the compute.sls, which they have shared, but since I don't really have a kilo system at the ready, I kept this in.

Once we're on a mitaka-only branch, we'll be able to cut these out.